.github/workflows/ci-pytorch-dockers.yml

```
@@ -37,6 +37,7 @@ jobs:
           - {python_version: "3.9", pytorch_version: "1.10", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.6.1"}
+          # TODO: add 1.13
```

```suggestion
          - {python_version: "3.9", pytorch_version: "1.13", cuda_version: "11.6.1"}
```
I think this shall be added, right?